actors/evm/src/interpreter/precompiles/mod.rs

```diff
@@ -104,8 +103,8 @@ impl<RT: Runtime> Precompiles<RT> {
 
     /// Checks if word is an existing precompile
     #[inline]
-    pub fn is_precompile(addr: U256) -> bool {
-        !addr.is_zero() && Self::lookup_precompile(addr).is_some()
+    pub fn is_precompile(addr: &EthAddress) -> bool {
+        !addr.is_null() && Self::lookup_precompile(addr).is_some()
```

Comment:
the is_null() check shouldn't be necessary anymore after https://github.com/filecoin-project/ref-fvm/issues/1474 since we will be sending all funds to be burned anyways, though its probably better to keep this seperate just for sensibility.

Reply:
I can adapt as needed during implementation, this works for now and i want to work off from here.